### PR TITLE
Update metadata.json to support GNOME Shell 47 (#488)

### PR DESCRIPTION
**Pull request by bayramkzk**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "shell-version": [
-        "46"
+        "47"
```
Comment (marmitar) [inline metadata.json:3]: If the code works with both versions, then why not use both?

```suggestion
        "46", "47"
```

**Comment by marmitar** [comment]
The extension is working well for me.

**Comment by NormTurtle** [comment]
please merge

**Comment by Tudmotu** [comment]
Thank you, uploaded to e.g.o, waiting for approval:
https://extensions.gnome.org/review/58775

**Pull request merged**